docs/Profiles.md

```diff
@@ -0,0 +1,36 @@
+# Profiles
+
+User profiles are represented as JSON objects that follow a [JSON
+schema](http://json-schema.org/).  The schema contains descriptions of its
+fields and is the primary reference for the content of a profile.  It is
+available at
+[cis/plugins/validation/schema.json](/cis/plugins/validation/schema.json).
+
+A [sample test profile](/tests/data/profile-good.json) is also available,
+although it omits many optional fields.
+
+## Groups
+
```

Comment:
I think this needs a picture and some rewording but would merge it as is for now, until we improve it